README.md

```diff
@@ -1 +1,68 @@
-Please refer to the official documentation [here](https://ob.docs.wso2.com/en/latest/)
+<!--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+# WSO2 Financial Services Accelerator
+
+Financial Services Accelerator a collection of technologies that increases the speed and reduces the complexity of adopting open banking compliance.
+
+### Building from the source
+
+If you want to build Financial Services Accelerator from the source code:
+
+1. Install Java8 or above.
+1. Install [Apache Maven 3.0.5](https://maven.apache.org/download.cgi) or above.
+1. Install [MySQL](https://dev.mysql.com/doc/refman/5.5/en/windows-installation.html).
+1. To get the Financial Services Accelerator from [this repository](https://github.com/wso2/financial-services-accelerator.git), click **Clone or download**.
```

Comment:
lets mention to select he relevant branch and then download

Reply:
Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/e755ebde386e6560d9acf08b41c55992ca7d6562